randovania/games/prime1/json_data/Magmoor Caverns.txt

```diff
@@ -1651,15 +1670,19 @@ Extra - aabb: [360.1392, -1048.9818, 21.99028, 492.0931, -921.3259, 113.2094]
           Any of the following:
               Space Jump Boots
               Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-              Scan Visor and Combat/Scan Dash (Intermediate) and Slope Jump (Beginner) and Standable Terrain (Beginner)
+              All of the following:
+                  # https://youtu.be/3N7BMPBBHsI at 0:12 turn around and use standable https://youtu.be/qnOw1NJdHDg?t=8
+                  Scan Visor and Combat/Scan Dash (Intermediate) and Slope Jump (Beginner) and Standable Terrain (Beginner)
           Knowledge (Beginner) or Heat-Resisting Suit
   > Door to Transport Tunnel C
       All of the following:
           Knowledge (Beginner) or Heat-Resisting Suit
           Any of the following:
               Space Jump Boots
               Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
-              Scan Visor and Combat/Scan Dash (Intermediate) and Slope Jump (Beginner) and Standable Terrain (Beginner)
+              All of the following:
+                  # https://youtu.be/3N7BMPBBHsI
```

Comment:
Keep in mind for the future, this duplicate video is a good example of why it's preferable to have more nodes with fewer connections than vice versa. (Not your fault)

Reply:
Understood, noted.